content/docs/how-tos/onboarding-a-new-component.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Onboarding a New Component for Testing and Merge Automation
+date: 2020-10-05T11:14:39-04:00
+draft: false
+---
+
+## Overview
+
+This document overviews the workflow for onboarding new public component repositories to the Openshift CI. Private
+repositories are not supported at this time, but will be in the future
+
+## Repository Configuration
+
+### Webhook Delivery
+
+Our CI system is event-driven from GitHub via webhook delivery; the system will only function if webhooks for all events
+are delivered from a repo or org. Today, we have organization-level webhooks delivered for all repos in the following
+orgs:
+
+* openshift
+* openshift-s2i
+* operator-framework
+
+
+If you are onboarding a new component in one of these orgs, you do not need to configure webhooks individually for your
+repository, please continue on to the next section.
+
+#### Token Retrieval
+
+Webhook deliveries to the CI system are encrypted with an HMAC token provided when configuring the webhook in the GitHub
+UI. A CI administrator can configure the webhook appropriately for your repository if you grant them edit access, or you
+can contact a CI administrator to get a copy of the token so that you may add the webhook yourself.
+
+Contact a CI administrator via the @dptp-helpdesk alias in the #forum-testplatform channel in the CoreOS slack.
+
+#### Webhook Setup
```

Comment:
Can we suggest using the `add-hook` binary if it's published in a container image?

Reply:
What is `add-hook`?

Reply:
A tool fejta wrote in test-infra to automate this process of adding the webhook

Reply:
That tool aparently morphed into `prow/cmd/hmac` and that requires access to the prow cluster: https://github.com/kubernetes/test-infra/blob/1b5f13772461c1200b87c050a91f97f1daa934bd/prow/cmd/hmac/main.go#L126

So I'll leave it as-is.

Reply:
Gotcha